test/unittests/terminology.cpp

```diff
@@ -0,0 +1,39 @@
+
+#include "../../../../mapping-gfbio/src/util/terminology.h"
```

Comment:
how did you execute this test? Currently, only tests in mapping-core are executed. Maybe you can try to look into including module tests in the same way as the module src is added to the core.

Reply:
Ah, I didn't add the comment, sorry. I was executing this class from core while testing, but copied it to gfbio. It is not executable at the moment but I wanted to keep the test somewhere.
I will look into adding the test.